src/main/java/database/service/entity/Course.java

```diff
@@ -0,0 +1,44 @@
+package database.service.entity;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Course extends BaseEntity {
+    private String name;
+    private final List<Student> students = new ArrayList<>();
+
+    public Course() {
+    }
+
+    public Course(int id, String name, List<Student> students) {
+        super();
```

Comment:
`id` field will be ignored here. Add new constructor in `BaseEntity` and use it as a super constructor.